### PR TITLE
add 119 release notes draft

### PR DESCRIPTION
**Pull request by codyjlandstrom**
Initial draft of 1.19 release notes - includes CLI

### CODE REVIEW

File: src/content/release-notes.mdx [hunk 1]
```diff
@@ -5,6 +5,48 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.19.0
+
+16 April 2024
+
+This versions is compatible with Kubernetes versions 1.25 to 1.29
```
Comment (codyjlandstrom) [inline src/content/release-notes.mdx:12]: @ifbyol confirming we are officially supporting 1.29 in this release

Comment (pchico83) [reply src/content/release-notes.mdx:12]: 1.29 is on beta on this release, and 1.25 should be marked as deprecated

File: src/content/release-notes.mdx [hunk 2]
```diff
@@ -5,6 +5,42 @@ sidebar_label: Release notes
...
+
+### New Features
+
+- Okteto Insights: Pod metrics v2  <!-- 7897, 7906 -->
```
Comment (pchico83) [inline src/content/release-notes.mdx:16]: Should we combine all of them in single line and link to the specific docs?

File: src/content/release-notes.mdx [hunk 3]
```diff
@@ -5,6 +5,42 @@ sidebar_label: Release notes
...
+- Okteto Insights: User metrics  <!-- 7898 -->
+- Okteto Insights: Namespace and Preview metrics  <!-- 7900 -->
+
+### Improvements
+
+- Okteto CLI: `okteto deploy` will now ignore host volumes. To manage the mounting of volumes, use a Dockerfile with the `COPY` instruction instead of references in your Docker compose.
```
Comment (pchico83) [inline src/content/release-notes.mdx:22]: This is a breaking change. We should mark it as a breaking change and link to the community post about how to overcome the breaking change

**Review by pchico83 — COMMENTED** [review]
@codyjlandstrom in general, I think we need to make the release notes more user friendly. For example, I don't think our users would understand `Fix endpoints links spanning the full width of container`

File: src/content/release-notes.mdx (hunk 1 above)
Comment (ifbyol) [reply src/content/release-notes.mdx:12]: AFAIK, we jsut did a change to start running tests in 1.29, but nothing else was modified. We show a warning in the installation, so I would say we don't support it. At least, not in the same way we did in the past

File: src/content/release-notes.mdx (hunk 3 above)
Comment (codyjlandstrom) [reply src/content/release-notes.mdx:22]: Followed breaking changes convention and made a section for this. Leaving this open until the community post goes live

File: src/content/release-notes.mdx (hunk 1 above)
Comment (codyjlandstrom) [reply src/content/release-notes.mdx:12]: just confirming. We should update this to be 1.26 to 1.28?

Comment (pchico83) [reply src/content/release-notes.mdx:12]: @codyjlandstrom 1.19 supports 1.29 in beta, and marks 1.25 for deprecation on the next release, but 1.25 is till supported.
Our policy is:
- Run all tests for a new kubernetes release
- Mark the new kubernetes release as beta support, and mark for deprecation the old one
- For 1.20, support for 1.25 is removed, and 1.29 is fully supported and the recommended one

Comment (mnevadom) [reply src/content/release-notes.mdx:12]: > @ifbyol we have run all the tests in all the different cloud providers, and staging and the product cluster are running in 1.29 for a while now. @mnevadom could you check why we are showing a warning for 1.29 and incorporate this change to our guides to support a new kubernetes version?

Normally we do it in two steps, first add the version to test in nightlies and then modify the Chart and if the old version is already removed, remove it from support. 
My understanding from your messages is that we should already add for this version the warning including 1.29, and once we release/cut this version, remove 1.25 right?

Comment (pchico83) [reply src/content/release-notes.mdx:12]: @mnevadom we support 1.29, we shouldn't show a warning for 1.129 anymore. And yes, once we release this chart version, support for 1.25 can be removed

File: src/content/release-notes.mdx [hunk 4]
```diff
@@ -5,6 +5,43 @@ sidebar_label: Release notes
...
+
+### Improvements
+
+- Okteto CLI: [Smart Builds implementation](core/build-service.mdx) now uses `build: context` instead of `project: commit`
+- Update buildkit to v0.12.5-okteto1  <!-- 7830 -->
+- Add Grafana endpoint to `cluster/general` to support Oketeo Insights for SaaS plan users  <!-- 7861 -->
+- maxNamespaces quota should no longer affect the okteto-bot user  <!-- 7905 -->
+- Rename OKTETO_CURRENT_DEPLOY_BELONGS_TO_PREVIEW to OKTETO_IS_PREVIEW_ENVIRONMENT and allow it to be used for destroy operations  <!-- 7904 -->
```
Comment (ifbyol) [inline src/content/release-notes.mdx:28]: In this one, I would say that we added the environment variable `OKTETO_IS_PREVIEW_ENVIRONMENT` for dev environments deployed from UI and previews. `OKTETO_CURRENT_DEPLOY_BELONGS_TO_PREVIEW` was something internal and we never document nor tell customers to use it

File: src/content/release-notes.mdx [hunk 5]
```diff
@@ -5,6 +5,43 @@ sidebar_label: Release notes
...
+
+- Okteto Insights: Pod v2, User, Namespace, and Preview metrics are now available <!-- 7897, 7906, 7898, 7900 -->
+
+### Breaking Changes
+
+- Okteto CLI: `okteto deploy` will now ignore host volumes. To manage the mounting of volumes, use a Dockerfile with the `COPY` instruction instead of references in your Docker compose.
+
+### Improvements
+
+- Okteto CLI: [Smart Builds implementation](core/build-service.mdx) now uses `build: context` instead of `project: commit`
```
Comment (pchico83) [inline src/content/release-notes.mdx:24]: @codyjlandstrom link is ready:
https://www.okteto.com/docs/1.19/core/build-service/#smart-builds

File: src/content/release-notes.mdx (hunk 4 above)
Comment (codyjlandstrom) [reply src/content/release-notes.mdx:28]: Ahh ok thanks for the clarification

**Pull request merged**